web_company_color/models/res_company.py

```diff
@@ -200,21 +201,27 @@ def _scss_get_sanitized_values(self):
         )
         return values
 
-    def _scss_generate_content(self):
+    def _scss_generate_content(self, uninstall=False):
         self.ensure_one()
         # ir.attachment need files with content to work
         if not self.company_colors:
             return "// No Web Company Color SCSS Content\n"
-        return self.SCSS_TEMPLATE % self._scss_get_sanitized_values()
+        if uninstall:
+            return (
+                self._get_scss_template_uninstall() % self._scss_get_sanitized_values()
```

Comment:
I tried to uninstall web_company_color but did not see any error. However, you are right, it might raise error without _get_scss_template_uninstall function here
 
 I also tried to update context from uninstall_hook and use super() instead of self, but uninstall fails saying super has no attribute 
![image](https://github.com/user-attachments/assets/e7e39f4e-6bcf-4fdb-b468-47fe56378b01)
It is trying to call the parent of web_company_color instead of the parent which needs to be called for module which is uninstalled